src/cityreader/cityreader.py

```diff
@@ -14,15 +25,27 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+
+
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+    with open(r"C:\Users\colin\Desktop\lambda-python\Sprint-Challenge--Intro-Python\src\cityreader\cities.csv") as csvfile:
+        readCSV = csv.reader(csvfile, delimiter=',')
```

Comment:
Awesome use of `with open()`. As we spoke about, cities.csv should have worked too.